htdocs/api/v0.0.3-dev/candidates/visits/images/headers/Headers.php

```diff
@@ -62,53 +62,66 @@ public function __construct($method, $CandID, $VisitLabel, $Filename)
      */
     public function handleGET()
     {
-        $TE = $this->getHeader('acquisition:echo_time');
-        $TR = $this->getHeader('acquisition:repetition_time');
-        $TI = $this->getHeader('acquisition:inversion_time');
-        $ST = $this->getHeader('acquisition:slice_thickness');
+        $te = $this->getHeader('acquisition:echo_time');
+        $tr = $this->getHeader('acquisition:repetition_time');
+        $ti = $this->getHeader('acquisition:inversion_time');
+        $st = $this->getHeader('acquisition:slice_thickness');
 
-        $SeriesName        =  $this->getHeader("acquisition:protocol");
-        $SeriesDescription = $this->getHeader("acquisition:series_description");
+        $seriesName        =  $this->getHeader("acquisition:protocol");
+        $seriesDescription = $this->getHeader("acquisition:series_description");
+
+        $xspace = [
```

Comment:
I think our standard is to use `array(` style syntax, not square brackets.

Reply:
corrected it in Headers.php and QC.php.